src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -54,13 +54,17 @@ const title: { [missionName in MissionInformation.MissionName]: string } = {
   uuvRescue: 'UUV Rescue',
 };
 
-type GeofenceChecklistType = 'geofenceTop' | 'geofenceLeft' | 'geofenceRight' | 'geofenceBottom';
+type GeofenceChecklistType = 'geofenceTop' | 'geofenceLeft' | 'geofenceRight' | 'geofenceBottom' | 'geofenceTop2' | 'geofenceLeft2' | 'geofenceRight2' | 'geofenceBottom2';
 
 const checklistCache: { [check in GeofenceChecklistType ]: number | undefined} = {
   geofenceTop: undefined,
   geofenceLeft: undefined,
   geofenceRight: undefined,
   geofenceBottom: undefined,
+  geofenceTop2: undefined,
```

Comment:
can we get better names for these? I think geofenceKeepInTop and geofenceKeepOutTop sounds better